newdle/client/src/components/UserAvatar.js

```diff
@@ -12,8 +12,8 @@ const defaultArgs = [
   true, // rounded
 ].join('/');
 
-function UserAvatar({user: {email, name, initials}, className, withLabel, size}) {
-  initials = initials.replace(/[/?#]/g, '');
+function UserAvatar({user: {email, name}, className, withLabel, size}) {
+  const initials = name && name.length ? name[0] : 'N';
```

Comment:
Don't we always have a name here?

Reply:
I suppose we do, but by default I was trying to be defensive.

Reply:
I don't think there's a decent fallback though :/ the random-ish char feels weird there.
Checking `name.length` isn't needed, empty strings are falsy.

Reply:
In the very unlikely event of having an empty string there, it will return `undefined`. The random-char was based on "N" from Newdle. But I can always ignore that case since I don't expect it to happen.

Reply:
yeah i'd go for ignoring it